tests/pkg/lwip/README.md

```diff
@@ -0,0 +1,139 @@
+# lwIP test application
+
+This application provides an example of how to use lwIP and provides test for over-the-air
+interaction between two lwIP nodes.
+
+## Testing on `native`
+
+### IPv6
+
+For IPv6 on `native` there is a test script. It requires for a TAP bridge to be set up between two
+interfaces. For that run
+
+```console
+$ sudo dist/tools/tapsetup/tapsetup
+```
+
+Then just run it with the usual test command.
+
+```
+$ make flash test
+```
+
+### IPv4
+
+For IPv4 a DHCP server is needed. If not already done for the IPv6 tests, please set up a TAP bridge
+now. Then add an IPv4 address to it, e.g.,
+
+```console
+$ sudo ip addr add 10.233.68.1/24 dev tapbr0
+```
+
+Then configure a DHCP server (e.g. [`dhcpd`](https://wiki.archlinux.org/title/Dhcpd))
```

Comment:
Maybe add a note that one could also use `tapsetup` with the `-u <interface>` option to bridge the virtual network with an existing one that might already provide a DHCP server

Reply:
Let's keep it simple. Not that people complain that they suddenly don't have network anymore because their upstream interface is in a (potentially) firewalled bridge ;-).